### ISSUE TITLE
install next to existing provider

### ISSUE THREAD

**Issue by keithfraley** [koopjs/koop-provider-google-analytics#2]
I am interested in installing this provider along side my existing provider. Is that possible, if so, how?

**Comment by rgwozdz** [comment]
Yes, it can be used in a koop instance that already has other providers.  Just register the provider as you do all others.  Then make the noted additions to your config file and environment variables.  You'll have to have a Google Analytics account to get the required config setting from Google Analytics.  You would also need to have telemetry set up on whatever application you'd like to report analytics for.  This provider only moves analytics reports from Google to Koop.  It doesn't send data to Google.
